### ISSUE TITLE
Set fix IP

### ISSUE THREAD

**Issue by mawi1986** [Qrome/printer-monitor#81]
is it possible to give the ESP8266 a fixed IP address?
If so, where can this be done?

**Comment by Qrome** [comment]
Yes, I usually do it in my router.   Get the mac address from the DHCP and tell your router to assign the IP you want dedicated to it.   Otherwise you can modify the code in the WifiManager section to assign the IP.   Personally, I like to manage my hard IP addresses in one place on my router.
